.eslintrc

```diff
@@ -18,8 +18,13 @@
       "files": [ "**/*.ts" ],
       "extends": [
         "eslint:recommended",
-        "prettier"
-      ]
+        "prettier",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended"    
+      ],
+      "rules": {
+        "@typescript-eslint/no-inferrable-types": "off"
```

Comment:
Ok, I just check. Changing this rule does not affect on anything you said. So, it is not needed. It is probably something else which replaces `function name()` with `name()`

Reply:
You are right, sorry the topic `function name()` and `name()` I got it at one stage with other rules. This one is related with the avoidance of inferrable types.

If you delete the line 26, and run `npm test` you will see that `fraction.ts` and `tuning.ts` get changes like this one:
```
numerator: number = 1; 
```
to 
```
numerator = 1;
```